### PR TITLE
Snowglobe Station - A new icemoon station for nova!

### PR DESCRIPTION
**Pull request by MrSunabouzu**
## About The Pull Request

A new station, a nova specific ice moon station called Snowglobe station! This is the first gamedev thing i've made in 4 years since shipping black mesa and it was a blast. I'd love to share it! The station 3 is z levels. z3 is the main station level. Almost all rooms and departments are on it, z2 is a sprawling basement full of rooms to loot or renovate, and amenities supporting rp interactions! The service hallway is modeled lightly after a shopping mall, and designed to be very friendly to navigate. To help facilitate traitors, ladders exist in every maint tunnel on z3 so a neer do-well can disappear into the labyrinth below.

This station implements no new assets or features as it is, there is a spot that was designed for a space elevator but was converted to a cafe. The only new things code side are areas and absolutely necessary config changes.

The planet side features are very generation heavy, I only use z3 for external infrastructure. The hope is that over time the addition of more ruins will make snowglobe a map that is fun to explore the mini-content others make. So I put aside as much space as possible for it!

I'm also quite proud of the name. Sphere shaped and on an ice moon, get it?

## How This Contributes To The Nova Sector Roleplay Experience

New maps are cool! And throughout my time playing a consistent wish i've seen from people in ooc is another snow map so they can roleplay their ice cats. I think a second snow planet station would let us justify making more things FOR that biome or supplementing anything that upstream makes for it. Also the dorms are meticulously designed so support roleplaying cuddling with someone next to a roaring fire while you watch a blizzard outside. I believe this to be peak.

## Proof of Testing

<details>
<summary>Screenshots/Videos</summary>
  

</details>

## Changelog

:cl:
add: Snowglobe Station, a new icemoon station
config: adds new areas to station_areas in modular_nova
config: adds snowglobe to maps.txt
config: adds snowglobe to automapper blacklist
/:cl:

### CODE REVIEW

File: config/maps.txt [hunk 1]
```diff
@@ -16,12 +16,12 @@ endmap
 # Production-level maps.
 
 map birdshot
-	#default
 	maxplayers 66
 	votable
 endmap
 
 map deltastation
+	#default
```
Comment (MrSunabouzu) [inline config/maps.txt:24]: oh. oops i forgot birdshot was default originally

File: config/maps.txt [hunk 2]
```diff
@@ -16,12 +16,12 @@ endmap
 # Production-level maps.
 
 map birdshot
-	#default
 	maxplayers 66
 	votable
 endmap
 
 map deltastation
```
Comment (Jolly-66) [inline config/maps.txt:23]: Make sure to revert/fix this.

**Comment by Iamgoofball** [comment]
did a double-take at that username, not gonna lie

**Comment by Zenitheevee** [comment]
Rid us of the dreaded Automapper.

Let the dawn of a fresh, Non-TG maps rise.

File: _maps/nova/automapper/automapper_config.toml [hunk 3]
```diff
@@ -464,3 +464,20 @@ directory = "_maps/nova/automapper/templates/wawastation/"
 required_map = "wawastation.dmm"
 coordinates = [63, 127, 2]
 trait_name = "Station"
+
+# SNOWGLOBE STATION MAP EDITS
+# Icecats Camp Lower Level - Snowglobe
+[templates.snowglobe_icecats_lower]
```
Comment (Iamgoofball) [inline _maps/nova/automapper/automapper_config.toml:470]: You should not be using automapper templates on Nova original maps. Just map the content in yourself.

Comment (MrSunabouzu) [reply _maps/nova/automapper/automapper_config.toml:470]: This is the solution agreed upon in the discord mapping thread, with the consultation of the server owner and zerg. It will not be hard baked.

Comment (Zergspower) [reply _maps/nova/automapper/automapper_config.toml:470]: It's fine, it's a ruin that's also on icebox and i'm not maintaining 2 versions of the same thing

**Comment by Iamgoofball** [comment]
As a reminder, come August, there is a very large visual redesign of the worldspace coming with the Wallening from the upstream, and I highly doubt you guys will be skipping it. Here's some WIP shots, not all assets have been moved to the new visuals yet(we've currently got a large bounty going for this at /tg/ to wrap it up in time) but come August 1st, we're shipping Wallening at /tg/ and it will come down here as a result, so you may wish to be ready to redesign your map to accommodate for this new visual style.

image_-_2024-06-06T120353 429
walleninglava

**Comment by Zergspower** [comment]
yeah we're ... thrilled... yay...

**Comment by Iamgoofball** [comment]
Yeah, it's gonna be a bit of an adjustment period and the maps are going to need a visuals pass, but we'll finally have a visual perspective that doesn't look like ass as a result.

**Comment by Zergspower** [comment]
ngl Shiptest did it right a while back, but this isnt the place to bitch about the list of things i hate about this new look. We have however started to look at our own maps and cried at the task ahead though

**Comment by Zergspower** [comment]
Been a while sorry about that - no energy at all with how busy work is

One tip i can give you is how to use the hide feature - in the environment tab the eyeball there if you click it will put checkboxes next to all the items.
image

Meaning you can hide entire groups such as objects, then select specifically which ones you want shown and it makes you smack your forehead often times on how easy it is to see it that way



<details>
<Summary>Disposals </Summary>
Shared Engineering Storage has a stray chute but no pipes
image

Engineering Security Post in a similar state
image

CE's Office is missing Disposal Pipes
image

Actually looks like Engineering was missed on the disposals run, so i'll let you manage that before rechecking.
image

Dorms Disposal chutes Orphaned
image

Botany - Try to avoid sending pipes through walls as often as possible as it makes repairing harder to do - but as shown you have an extra pipe and trunk underneath the one placed
image

</details>

<details>
<Summary>Atmos - these are important as unconnected pipes will make angry CI noises - plus the needing to breathe part </Summary>
Botany is missing scrubbers

image

SM Room - having distro and waste running through the entire room is generally not a good idea as it makes repiping fairly difficult, and currently your waste is hooked into the SM's waste as well
image

Scrubber outside of Science will need some sort of pump to push it into waste 
image

Maints above atmos seem to have pipes leading places but then dont connect
image

image

image

image

image

image

Maints above Psych similar issue
image
image

Abandoned Wrestling Arena
image

Abandoned Cafeteria
image
image

Maints above Holodeck
image

Outside Holodeck 
image


</details>

**Comment by Zergspower** [comment]
Any specifics to why the raptor pen is so far away?

And can that launcher for Ord travel that distance?

**Comment by Zergspower** [comment]
Son of a bitch hold on Github desktop lied to me

**Comment by MrSunabouzu** [comment]
> Any specifics to why the raptor pen is so far away?
> 
> And can that launcher for Ord travel that distance?

Oh its far away just because I thought itd be a good reward for exploring the otherside of what i lovingly call the Ankle Destruction Cliff

And yeah i yote a banana up to the detonation chamber unless ttvs go less distance on a driver

**Comment by Thlumyn** [comment]
Hi, I've got a few comments as well, just as a fellow mapper

<details>
<summary>Atmos</summary>
It kind of feels like the crystallizer is an afterthought here: Like, the HFR gets its own room and floor design and then the crystallizer is just there. Maybe add some floor decals around it or something to make it look more like something mapped in vs something added later by an atmos tech. 

image

It looks like your gas tanks are missing the tank control computers, and the gas sensors (though you have the input vents, so theres no computer to control them).

image

What is going into the mix input? I like the general idea of having atmos being a bit more free form like this, but it still looks weird to just have an injector going into nothing but an r wall, maybe you should have a pipe into pipe layer there to match the yellow ones? 

image

I've got some concerns about this distro output here. 
Firstly, having the distro/scrubbers pipes under that Air to Distro pump will probably be a little annoying, because that looks like it would be the idea place to add in a layer adapter -> valves/pump -> layer adapt set up that atmos techs add to optimize distro. Same concern with that layer adapter right under it; that would be disconnected if that section were removed to add that. Also, it seems to be missing a thermo in the distro output (which, especially on an ice station is pretty important, as some people use the thermo to slightly overheat distro to counteract temperature loss. Also, there is no pump going into the waste loop.

image

About that waste loop: 

It's usually nice to have a shutoff before they get filtered into gasses. Like, when the gasses get really hot, I like to turn on that so that we aren't filter overheated N2 and O2 back into the gas tanks which can lead to a hotter distro than expected. 

For example, ice box has it here: 
image

The way this waste disposal loop has a valve from a planetary turf going into these filters means that when left alone, if that valve is open, it will just keep grabbing and filtering cold air from outside, put the extra cold O2 and N2 into the storage tanks and then adding the rest - probably cold CO2 into the general waste loop.... looking into it I probably should change this on my map to something like a pressure valve as well, o o p s

image

Atmos maints doors are general maints access so anybody can enter atmos 

image

Even if you plan for this to be replaced and hooked into custom pipes every time, you should probably add a port or something there so it doesn't complain about one way connections. 

image

HFRs usually have heavy shutters around them, not regular shutters, they'd burn right through regular shutters (and tbh probably heavy ones as well, but...) 

image
</details>

<details>
<summary>Engi</summary>
Bottom engi guard door has regular engi access 

image

Why is this multilayer hub here? 

image

With the emitters being on the SMES output it might be problematic, both when the engine is starting (as they wont be getting any power to that cable until its already running) and if somebody were to upgrade the SMES and max input, it'd take the extra power that the emitters would be using. Ik there's been some issues recently with the cell changes causing brownouts and this might have been to avoid that, but hopefully the megacell PR should fix that 

image

I know Granite said to move the pipes away from the center of the engine, and I think this is definitely an improvement, but its still a bit off to have them running under the windows like that. If possible, might want to change? Since removing pipes under windows can be a pain, Im sure you know that as an engi main. 
Also those windows should probably have plating under them and not r floor. 

image

Nice. 

image

Might want to compare with other maps, the engi lobby doors and some other doors in engi usually have a combination of either engi or construction access and yours just have engi. 

image

More engi doors that are just general maints access. On nova assistants get maints access for all departments, so they'd be able to enter engi from maints. 

image

Missing access on tcomms doors. 

image

Tcomms isn't cooled... interesting. I'm assuming that a conscious decision there to make working on tcomms easier; but it also makes it easier to antags to break into tcomms as well. I advise putting a holopad in tcomms since AIs often check in during comms outages to bug the engineers working on it (I don't think they can stand not talking for a few minutes) 

Also, that's a nice looking cooling loop you have, interesting design. 

</details>

I'm still looking through the other departments

**Comment by Thlumyn** [comment]
<details>
<summary>Service</summary>
Botany missing firelocks. Also it is missing a grinder. 

image

With all the trays right next to each other, it makes it hard to control cross pollination. You might want to add windows/windoors between them, or separate them a bit so that they are not all touching. Void and blueshift are good examples of windows between the trays. 

image

With the kitchen, it'd be nice if there was something like donk ponkets or a cake batter to hold people over for a bit as the chef preps. 

Maints access directly into the kitchen 

image

Kitchen is missing lockdown shutters. 

You might want to add a windoor here so it at least has the illusion of being a librarian only desk. 

image

Chapel Storage has crematorium access (not chapel office?) and I dont see any crematorium (the item) 

image

This door here can probably be normal maints access and not chapel office 

image

Why make this door theatre access if anybody can walk down the stairs to get there? 

image

</details>

<details> 
<summary>Cargo</summary>
Might want to switch this computer and bounty pad - those conveyors might just grab up whatever somebody is trying to set down for a bounty, and not the cubes. 

image

Where are the buttons to control these shutters?

image

This door should probably be mining, if the others are mining. 

image

They have a mech but no power port 

image

Kheiral cuffs are a nice touch, but... isn't all the mining on station z levels? So suit sensors would work anyway

image

I think these outdoor paths are just adorable

image


</details>

Aaaannnd that's all the energy I have tonight. I'll try to look at more departments and mapping stuff tomorrow, or thursday

**Comment by Thlumyn** [comment]
<details>
<summary>Xenobio</summary>
Xenobio is missing some things: plasma, a grinder, a water tank + fire extinguishers (for rogue slimes), monkey cube boxes

Any reasons why the air alarms in there have the mixing chamber access helper? 

image

Also, there's no door anything between that science hall and the xenobio room so anybody with general sci access can get in. 
</details>

**Comment by MrSunabouzu** [comment]
> Xenobio
> Xenobio is missing some things: plasma, a grinder, a water tank + fire extinguishers (for rogue slimes), monkey cube boxes
> Any reasons why the air alarms in there have the mixing chamber access helper?
> 
> image
> 
> Also, there's no door anything between that science hall and the xenobio room so anybody with general sci access can get in.

Good catches, all of these posts! thank you so much for looking over snowglobe! Implemented all the suggested changes except the chapel crematorium access, because i put the crematorium in there o o p s

As for the lack of tcomms cooling, that could bite me in the ass if upstream decides temperature control matters. My solution honestly would be some passive vents that are exposed to the outside and heat radiator pipes on the inside. Cheap, cools it but kind of badly so it can be improved on, funny osha-violating design.

Lastly the air alarm in xeno, its like that since to my knowledge in code that means "if they are a scientist, let them use it" and that last 5x5 chamber is designed to be flooded with a specific gas of the scientists choosing, probably bz. The air alarm is how I intend them to inject the gas in there, just turning it on via alarm. ill admit its whimsical but theres a scrubber right there so how bad can they fuck it truly?

**Review by Jolly-66 — CHANGES_REQUESTED** [review]
Cursory review of some things I've noticed:

Go over every non-maint door and double check for firelocks. I found some doors without firelocks.

For departments such as Security. Engineering and Science, please double check your r-wall usage (or rather, security). Walls that face maintenance or the main hall NEED to be secured. Internal walls, walls that are used INSIDE the department, do not need to be secured. Rooms like the SM chamber, Ordnance, Head Offices and XB are exempt from this room for obvious reasons.

The same applies to windows. Reinforced windows need to be used on the outside. Usage of plasma glass should also be replaced with those windows (except in the SM chamber).

The clown and mime being spawned all over the station, on paper would be fine. But whenever we get this map ready for testing, clown/mime players are going to be disoriented from where they're starting (not to mention there is a clown spawn in maintenance!!). Keep them to the theater, backstage or even the bar/kitchen.

All dept sec checkpoints need to be devoid of sec stuff (records consoles). Department related consoles are fine however.

Double check window spawner tile turfs. There needs to be plating underneath them.

Double check the mapping verbs (active turfs, display intercom range) to make sure they're complicit.

**Review by Jolly-66 — CHANGES_REQUESTED** [review]
Some other quick nits before I inspect departments.

<details>
<summary> Nits </summary>

Please remove the light switches from the hallways. It realistically makes no sense. I know I told you to check the mapping tools, but you *really* don't need a light switch everywhere.

The plasglass/titanium tables need to be swapped out with any other table type. Regular reinforced/glass is fine, but no higher.

The chaos emerald lizards descriptions need to be changed. They feel memey and OOC. I'll accept them if their desc are based off of their names and aren't AS memey. A joke or two is perfectly fine.

There are also just, a TON of windows. Everywhere.

The bomb site is missing its camera (indestructible). I know its the bomb site, but spruce it up, it feels incomplete.
Also you really should put a *lot* of padding around this. A good max cap may destroy a nearby ruin and cause issues (if people are there)
image

Swap the bathroom doors out with generic ones, and add door lock buttons inside the stalls.
image

Tcomms AND tech storage need to be integrated better. They appear to be very shoe-horned in. Secure storage in tech storage can also be VERY easily broken in.
image
image

I know they work, but it looks weird. Place the plastic flaps ontop of the chutes.
image

These are the same console code-wise.
image

This entire section of maint needs to be redone and fleshed out more. Its essentially nothing but straight lines and bends.
image

Remove SOME of the shipping containers in exchange for more crates and random nonsense.
image

For these abandoned rooms, try to make them more derelict. Its strange having most of the machines intact.
image

The beds here feel very out of place, please remove them.
image

Coroner's office needs secure morgue trays.
image

Rotate these bedsheets.
image

This pipe is not `visible`.
image

The burn/freeze chambers need their unique areas.
image

Make a subtype or use a different turf, these aren't actually cold turfs.
image

Rejiggle the freezer. The freezer shouldn't be in maint like this, its only asking for trouble.
image

The chapel should have innate access to the burial grounds.
image

We don't need two mining consoles.
image

Where is cytology?
image

Clean up the siding here.
image

The 2x2 wall of hell. Readjust this.
image

Changing room could use a door lock, and move the curtains.
image

Flavorful, but use regular candles. Feel free to throw in an extra box or two, as a treat.
image

Please remove this cat bed in atmos, its very out of place.
image

The atmos tanks need to be connected to the main pipe net roundstart.
image


</details>

**Comment by Iamgoofball** [comment]
I do want to comment a bit more on this room in particular as it's very much an issue all of our downstreams have with mapping

346301592-e139ed2d-e0c4-4673-964d-785e5d559cd7

This room has absolutely zero reason to exist in the in-universe conceit that this is a corporate workspace by a corporation for people doing the daily SS13 shift at. It only makes sense in the conceit of this being furry meet 'n fuck kingdom 13.

A lot of the downstream maps have this issue, and the only map I've seen pull it off convincingly is Blueshift because Blueshift is designed like a cruise ship and it's largely constrained to the heads of staff offices which are expected to be opulent, the promenade on the left side of z1 that's basically a bunch of empty businesses, and the permabrig, which the eternal joke about it being a retrofitted resort section for rich people turned into a prison due to its isolated nature makes immensely funny.

Try to remember, you're building a functional workspace that's, as of the shift starting, built to corporate spec to a T(except with corners cut where possible to save on money, because Nanotrasen is nothing but cheap); why would corporate spec include such a room?

**Comment by Iamgoofball** [comment]
>Because its for an rp server. And sometimes people want to collaboratively write in an rp server in a room that's marginally harder to access so they are less likely to be disrupted by unwanted mechanical disruptions. Why would I design a station map for upstream and not the server I play on? Like I don't really care what the lore is on upstream. Its not for it.

Because it is an RP server, the station should make sense as a workspace especially because a high roleplay server like Nova Sector is expected to be realistic. It's out of place on a conceptual level because the kayfabe of the roleplay is that you're a bunch of people working on a space station, built by a corporation for people to work at. 

The Dormitories exist on an IC level because Nanotrasen understands their crew needs their privacy from time to time, allowing them to implicitly acknowledge "our crew fucks" without outright acknowledging that it's a common occurrence because this isn't Furry Meet N' Fuck Kingdom 13, this is Space Station 13.

The Dormitories also exist on an OOC and game design level because players be fuckin' and because players need a room they can bolt with plausible deniability to do bad guy things like assemble devices or buy traitor items off their uplink.

Nanotrasen very much exists in whatever exists as Nova's lore after the coup, and they very much still are the corner-cutting corporation that they are at /tg/ and at bay and at goon and at every single other server, because the core premise is still Space Station 13, it's why everyone's here and not at many of the other venues for collaborative roleplaying and writing. People wanna play SS13 because they wanna be immersed in SS13 while writing roleplay. It doesn't make sense in any stretch of the imagination for Nanotrasen to build public sex rooms for sex directly off of public hallways in their stations, or to haphazardly jam a bunch of beds into random rec-rooms for people to have sex on that are visible from the hallway

The issue that I have with this room is that it is not trying to be even remotely sneaky about this being the sex room for people to have sex. It could be louder about it, sure, but it's also not being quiet about it. It's not a side room that's out of the way and thus hard to find, it's hooked up to what appears to be a service hallway based on the green flooring with glass double-doors. If it were not for the presence of the pillow pile and the double-double beds, this would be a perfectly serviceable rec room and nobody would bat an eye. However, it's got a set of double-beds visible by someone walking by the doorway that don't really seem to have any purpose in the room.

Basically, it's the difference between a player going "oh, that's a neat place to have sex in that I just found" and "oh, that's the sex room, its for sex". One's cool and neat and makes the player feel like they found something cool. One's just checking a box on the list of things players expect to find on Furry Meet N' Fuck Kingdom 13.

**Comment by Iamgoofball** [comment]
Minor nitpick I noticed while loading the map up in the editor to draft a revised version of that room to elaborate on my concerns and to provide constructive criticism:
StrongDMM_fKN9Cua1LD

The Gulag is not set up properly; it's not using the unique gulag area or generation-turfs so megafauna, mining fauna, and ruins can spawn in the gulag's area. Additionally, people can just mine straight down if they get lucky with RNG and reach the stairs up on the Z-level to get back to the station with ease. Please check out how Ice Box does the gulag with the turfs and areas and the plasma river moat, as escaping the gulag is supposed to be a challenge that requires game knowledge of how to deal with the plasma/lava and shouldn't involve running into megafauna right outside the front door.

I did want to give props to this though: the little chair sitting at the ore processing center is hilarious and is a great example of the good worldbuilding and attention to detail I'm used to seeing in your past mapping projects in other games.

StrongDMM_9pZzGB0FCY

**Comment by MrSunabouzu** [comment]
> > Because its for an rp server. And sometimes people want to collaboratively write in an rp server in a room that's marginally harder to access so they are less likely to be disrupted by unwanted mechanical disruptions. Why would I design a station map for upstream and not the server I play on? Like I don't really care what the lore is on upstream. Its not for it.
> 
> Because it is an RP server, the station should make sense as a workspace especially because a high roleplay server like Nova Sector is expected to be realistic. It's out of place on a conceptual level because the kayfabe of the roleplay is that you're a bunch of people working on a space station, built by a corporation for people to work at.
> 
> The Dormitories exist on an IC level(and the goofy ass sex vendor that really could use a visual redesign pass to be more low-key) because Nanotrasen understands their crew needs their privacy from time to time, allowing them to implicitly acknowledge "our crew fucks" without outright acknowledging that it's a common occurrence because this isn't Furry Meet N' Fuck Kingdom 13, this is Space Station 13.
> 
> The Dormitories also exist on an OOC and game design level because players be fuckin' and because players need a room they can bolt with plausible deniability to do bad guy things like assemble devices or buy traitor items off their uplink.
> 
> Nanotrasen very much exists in whatever exists as Nova's lore after the coup, and they very much still are the corner-cutting corporation that they are at /tg/ and at bay and at goon and at every single other server, because the core premise is still Space Station 13, it's why everyone's here and not at many of the other venues for collaborative roleplaying and writing. People wanna play SS13 because they wanna be immersed in SS13 while writing roleplay. It doesn't make sense in any stretch of the imagination for Nanotrasen to build public sex rooms for sex directly off of public hallways in their stations, or to haphazardly jam a bunch of beds into random rec-rooms for people to have sex on that are visible from the hallway.
> 
> The issue that I have with this room is that it is not trying to be even remotely sneaky about this being the sex room for people to have sex. It could be louder about it, sure, but it's also not being quiet about it. It's not a side room that's out of the way and thus hard to find, it's hooked up to what appears to be a service hallway based on the green flooring with glass double-doors. If it were not for the presence of the pillow pile and the double-double beds, this would be a perfectly serviceable rec room and nobody would bat an eye. However, it's got a set of double-beds visible by someone walking by the doorway that don't really seem to have any purpose in the room.
> 
> Basically, it's the difference between a player going "oh, that's a neat place to have sex in that I just found" and "oh, that's the sex room, its for sex". One's cool and neat and makes the player feel like they found something cool. One's just checking a box on the list of things players expect to find on Furry Meet N' Fuck Kingdom 13.
> 
> There is a lot of very neat and very cool things about this map; I think overall it's a massive improvement on the planetary station design compared to Ice Box, which was our first attempt at a planetside map on /tg/ and has remained for ages mostly untouched outside of the few changes over the years. I just think it needs a little more polish to really fit in with the map lineup and the kayfabe of SS13 as a roleplay setting where people have sex occasionally.

wait do you really think i expect that to be the big erp room? the door doesn't even bolt, no one would want to do it there. its for like, if you legitimately want to continue an ic conversation. like this happens a lot to me on my time in rp servers, you find out your character and someone else's share similar ideals and the vibes are fun and you get to know each other via rp. like. platonically. "oh hey your character has the same views as mine on silicon rights? cool lets find a spot to talk" the quiet room or the library are the intended spots for such an activity to pan out. The beds are if your like. a caregiver irl and expect to be afk for 20 or so minutes but don't want to or cannot retreat to a dorm to idle. you go here, the place you park your funny sprite in the spaceman game. Hell even on goon there's a couple useless spaces on nadir and donut 2 that are just "yeah go here to talk privately". It didn't even occur to me until you pointed it out that this could be a focal point for public sex and like. yeah. it can. depends on the round and the server and the phase of the moon. Nova swings wildly between horny and not horny. 

That said, the nightclub is absolutely designed for what you just described. That's why it has an empty room where someone can put down in big bold letters "dont open horny inside". There's really no reason to erp in here when the nightclub is right there unless your like, willfully negligent of 18+ rules. The fact that is accommodates traitors and gives antags a blind spot to openly plot and shit is a side effect but pretty funny.

The next part i will say with my entire chest: the pillows are so my original character can cuddle atop their giant dragon girlfriend and just not do anything for about 2 hours while they enjoy small talk with her else instead of do their job. It's cute as fuck to roleplay mundane domestic bullshit. I am a villain in this way and do not repent.

**Comment by MrSunabouzu** [comment]
> Minor nitpick I noticed while loading the map up in the editor to draft a revised version of that room to elaborate on my concerns and to provide constructive criticism: StrongDMM_fKN9Cua1LD
> 
> The Gulag is not set up properly; it's not using the unique gulag area or generation-turfs so megafauna, mining fauna, and ruins can spawn in the gulag's area. Additionally, people can just mine straight down if they get lucky with RNG and reach the stairs up on the Z-level to get back to the station with ease. Please check out how Ice Box does the gulag with the turfs and areas and the plasma river moat, as escaping the gulag is supposed to be a challenge that requires game knowledge of how to deal with the plasma/lava and shouldn't involve running into megafauna right outside the front door.
> 
> I did want to give props to this though: the little chair sitting at the ore processing center is hilarious and is a great example of the good worldbuilding and attention to detail I'm used to seeing in your past mapping projects in other games. The camera directly above the beds with the privacy curtains is also a nice touch.
> 
> StrongDMM_9pZzGB0FCY

oh shit nice thanks this is just straight up useful. SDMM is a really intuitive tool but this is still my first map in is so I fucked up a lot of small specifics. I've never seen anyone use the gulag on an rp server so who knows how long this wouldve been busted.

**Comment by Iamgoofball** [comment]
I hate to say this but a regular issue at Skyrat before the coup to Nova was people turning the whole ass Bar into a strip club and doing extremely public sex., so admittedly I've got some bias and PTSD from players being absolute dumbshits in relation to this, but yeah, fair, my bad for making assumptions about the reasoning for the beds, that makes more sense but at the same time makes me wish we had like, a recliner chair or a pool-side lounge chair style thing you could put there instead, as it'd communicate that way, way better than the full ass double-beds with sheets imho

>The next part i will say with my entire chest: the pillows are so my original character can cuddle atop their giant dragon girlfriend and just not do anything for about 2 hours while they enjoy small talk with her else instead of do their job. It's cute as fuck to roleplay mundane domestic bullshit. I am a villain in this way and do not repent.

amen, respect the hustle, this is god's work

**Comment by MrSunabouzu** [comment]
> I hate to say this but a regular issue at Skyrat before the coup to Nova was people turning the whole ass Bar into a strip club and doing extremely public sex., so admittedly I've got some bias and PTSD from players being absolute dumbshits in relation to this, but yeah, fair, my bad for making assumptions about the reasoning for the beds, that makes more sense but at the same time makes me wish we had like, a recliner chair or a pool-side lounge chair style thing you could put there instead, as it'd communicate that way, way better than the full ass double-beds with sheets imho
> 
> > The next part i will say with my entire chest: the pillows are so my original character can cuddle atop their giant dragon girlfriend and just not do anything for about 2 hours while they enjoy small talk with her else instead of do their job. It's cute as fuck to roleplay mundane domestic bullshit. I am a villain in this way and do not repent.
> 
> amen, respect the hustle, this is god's work

Naw its cool dude its a touchy subject, nova's ruleset is structured in a way so that consent of the audience is considered. You do it in rooms with boltable doors or in a space that is clearly designed for private matters and can be marked in some way as "don't go in". A lot of people use the labeler or whatever to put "do not enter" on doors then weld it. Blueshift has something like this with its backroom but I don't think it went far enough, hence a changing room plus 2 3x3 booths to outright accommodate sex work in a way that isn't embarrassing. I've ended up very ride or die for erp in general because I believe it to be an excellent tool for gender exploration in general and I think the space deserves a modicum of dignity. It can be a really healing creative writing exercise under the right circumstances. My earnest hope is a lot of very small private communities steal my map to explore themselves in a natural and safe way. 

Oh yeah relatedly apparently jazztronauts ended up fueling a few furry circles in vr chat creatively so i'm basically realizing im a lgbt johnny appleseed and i think it rules.

File: _maps/map_files/Snowglobe/snowglobe.dmm
Comment (GoldenAlpharex) [inline _maps/map_files/Snowglobe/snowglobe.dmm:1]: Here's a few preliminary things I noticed, it's not all of my feedback, but doing stuff in this format is pretty slow so I figured I'd send this now, at least.

<details>
<summary>Courtroom</summary>

image

I believe we usually have some intercoms on each side of the courtroom, so that the court case can be played over the radio.

</details>

<details>
<summary>Surgery rooms</summary>

image

Usually, we make it so they can have a way to make their windows opaque, to grant patients privacy during operations.

Furthermore, I'd probably move the smartfridge down one tile, and then turn the directional windows into full-tile windows, otherwise I feel like it looks a little awkward with the door being right next to it.

</details>

<details>
<summary>Labor Shuttle Hallway</summary>

image

It's surprisingly low in decals, it makes it look very plain and boring. I'd spruce it up a bit with some stuff around the edges, like most of the other rooms, at least. Same goes for the Corrections Officer Lounge that's just a little North from this screenshot's top, and in general, Perma and even Security as a whole need some sprucing up too, it's very monochrome and bland in there, as opposed to many other parts of the station.

</details>

<details>
<summary>EVA Storage</summary>

image

There's a weird wall on the outside, also there's a general lack of decals inside the room too.
</details>

<details>
<summary>Arrivals Shuttle computer in Bridge</summary>

image

I'm not really sure why there's an arrivals shuttle computer on the bridge, there's not really any use for it being there, you can't even see the arrivals shuttle from the bridge.

</details>

<details>
<summary>Bridge</summary>

image

I'd move the entire bridge here one tile to the right, so it actually aligns with the hallway that leads to it. It messes with my OCD that it's off-centered, and it doesn't look like it has too much of a reason to be off-centered.

</details>

<details>
<summary>Law Office Benches</summary>

image

They probably shouldn't be shifted up, it looks a little awkward because it clips into the wall and doesn't actually pixel-shift people up when they sit on them, so it looks like they're not really sitting on the bench.

</details>

<details>
<summary>Vents/Scrubber Placements</summary>

image

As you can see in the upper example, a lot of vent and scrubbers seem to be placed around randomly, which make the rooms look a little messy, I'd recommend trying to make things look a bit more intentional (trying to go for symmetry, putting them generally in the middle of rooms/hallways, makes more sense visually and actually usually makes them work better).

</details>

<details>
<summary>Orderly's Office</summary>

image

I'd probably add more windows around the airlock, otherwise you basically don't see anything in there.

</details>

**Review by GoldenAlpharex — CHANGES_REQUESTED** [review]

**Review by spockye — CHANGES_REQUESTED** [review]
Nice station, good work!

Couple of errors i noticed while looking it over

<details>
<summary>errors</summary>
<br>

image
missing supply pipe (z2)

image

Missing supply pipe (Z2)


image
supply unconnected from the grid (Z2)

image

this isnt connected to anything. unsure if thats intentional


image

seems like this was supposed to be connected? (z3)
</details>

**Comment by sqnztb** [comment]
Played a bit over the weekend. Botany issues:
- The right windoors inside botany's tray quads require cargo access to open and shut. 
- The windoors leading down into the service hall require `service` access which means anyone from service can just come in the back. They should have `hydroponics`.
- Same for the kitchen.
- And the bar. 
- Additionally I'd like the desk windoors flipped from the south edge to the north, that way botanists can put stuff on the table and people can actually take the items off said table without you needing to leave the windoors open (giving access to botany to any table jumper).

Botany-specific issues aside: The request console in botany is an `east` one that is manually pixel shifted back to x0 and y-33. The CE's request console is the same. And the HOS's. And several others. Please don't pixel shift directionals like this. The whole reason the directionals exist is so you don't have to. Please audit all of these.

**Comment by MrSunabouzu** [comment]
> Played a bit over the weekend. Botany issues:
> 
> * The right windoors inside botany's tray quads require cargo access to open and shut.
> * The windoors leading down into the service hall require `service` access which means anyone from service can just come in the back. They should have `hydroponics`.
> * Same for the kitchen.
> * And the bar.
> * Additionally I'd like the desk windoors flipped from the south edge to the north, that way botanists can put stuff on the table and people can actually take the items off said table without you needing to leave the windoors open (giving access to botany to any table jumper).
> 
> Botany-specific issues aside: The request console in botany is an `east` one that is manually pixel shifted back to x0 and y-33. The CE's request console is the same. And the HOS's. And several others. Please don't pixel shift directionals like this. The whole reason the directionals exist is so you don't have to. Please audit all of these.
> 
> Edit: and the dead body placers in the morgue. You only need ONE. Each placer adds 3 bodies to morgue trays and with 3 placers, you now have 9 bodies shoved into 6 trays. Remove the extras.

You are objectively right about the service tweaks so i've just done them and am about to upload it. Thanks! A few other departments would benefit from having two sets of windoors so I did that too.

Good catch on the request consoles, i copied them from other maps and just deleted and replaced each with an auto-name of the correct direction. Also removed two morgue markers.

**Review by GoldenAlpharex — APPROVED** [review]
It's been test-merged long enough, let's go and merge it!
